### DISCUSSION TITLE
如何添加 条件表达式

### DISCUSSION THREAD

**Discussion by cfc3058353263** [didi/LogicFlow#1252] [Q&A]
如何添加conditionExpression 条件表达式
<sequenceFlow id="Flow_1imp5md" name="" sourceRef="start_event" targetRef="Activity_1pwf50p">
            <conditionExpression xsi:type="tFormalExpression">111</conditionExpression>
</sequenceFlow>

**Comment by towersxu** [comment]
https://github.com/didi/LogicFlow/issues/1253
